CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+## TBD
+
+### Enhancements
+
+* Add option to configure what file paths are included in the project stacktrace (`vendor_path`)
```

Comment:
This doesn't seem clear, as all the file paths are usually included in the stacktrace. I would reword this to something like "Add option (`vendor_path`) to adjust filename matching for marking stackframes as in-project".